scenarios/README.md

```diff
@@ -0,0 +1,50 @@
+# Scenarios
+
+This folder contains various projects conducted in cyclus.
+
+## 2014-E&S
+Summary: Transition scenarios for evaluation groups from 
+the 2014 E&S Study from the U.S. Department of Energy.
+
+
+## 2017-bae-europe
```

Comment:
I think this project is from Jin Whan's MS thesis. Maybe link to that publication?

Reply:
I'll check it out!

Reply:
It's at least part of it! (I added a related to section)